pkg/internal/transform_test.go

```diff
@@ -0,0 +1,86 @@
+package internal_test
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/buildpacks/scafall/pkg/internal"
+
+	h "github.com/buildpacks/pack/testhelpers"
```

Comment:
We should not have circular dependencies between pack and scafall. scafall should be unaware of pack. This will cause issues down the road when pack imports scafall.

Reply:
Thanks.  I've switched to testify's asserts and I've broken the circular import.